content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -422,7 +422,7 @@ Helper programs relating to the volume type may be required for consumption of a
 
 ### Capacity
 
-Generally, a PV will have a specific storage capacity.  This is set using the PV's `capacity` attribute.  See the Kubernetes [Resource Model](https://git.k8s.io/community/contributors/design-proposals/scheduling/resources.md) to understand the units expected by `capacity`.
+Generally, a PV will have a specific storage capacity.  This is set using the PV's `capacity` attribute.  Read the glossary term [Quantity](https://kubernetes.io/docs/reference/glossary/?all=true#term-quantity) to understand the units expected by `capacity`.
```

Comment:
```suggestion
Generally, a PV will have a specific storage capacity.  This is set using the PV's `capacity` attribute.  Read the glossary term [Quantity](/docs/reference/glossary/?all=true#term-quantity) to understand the units expected by `capacity`.
```

Reply:
Glossary is too brief. It does not explain how to use it.
The archived doc will still be there. Can you change to the archived link instead: 
https://github.com/kubernetes/design-proposals-archive/blob/main/scheduling/resources.md

Reply:
@xing-yang we'd be happy to consider a _separate PR_ that adds a new page explaining what a quantity is, in more detail.

This PR was a reasonable specific fix.

Reply:
Also, we don't want to add new links to archived design proposals.